README.md

```diff
@@ -59,6 +59,12 @@ Unfortunately the Cassandra project does not always increment the `cqlsh` versio
 release we need to document not only the `cqlsh` version but also the `cassandra` version in which it
 shipped.
 
+#### 6.1.0 (Unreleased)
```

Comment:
This line needs updating from `Unreleased` to whatever day we release it.

Alternatively `git` tags / GitHub releases / PyPI will all keep track of release dates, so we could stop having them as part of the readme... but let's do that later in another PR, out of scope for this PR...